src/System.Management.Automation/engine/COM/ComUtil.cs

```diff
@@ -57,26 +57,9 @@ internal static string GetMethodSignatureFromFuncDesc(COM.ITypeInfo typeinfo, CO
 
             for (int i = 0; i < funcdesc.cParams; i++)
             {
-                COM.ELEMDESC ElementDescription;
-                int ElementDescriptionArrayByteOffset;
-                IntPtr ElementDescriptionPointer;
-
-                ElementDescription = new COM.ELEMDESC();
-                ElementDescriptionArrayByteOffset = i * ElementDescriptionSize;
-
-                // Disable PRefast warning for converting to int32 and converting back into intptr.
-                // Code below takes into account 32 bit vs 64 bit conversions
-#pragma warning disable 56515
-                if (IntPtr.Size == 4)
-                {
-                    ElementDescriptionPointer = (IntPtr)(ElementDescriptionArrayPtr.ToInt32() + ElementDescriptionArrayByteOffset);
-                }
-                else
-                {
-                    ElementDescriptionPointer = (IntPtr)(ElementDescriptionArrayPtr.ToInt64() + ElementDescriptionArrayByteOffset);
-                }
-#pragma warning restore 56515
-
+                COM.ELEMDESC ElementDescription = new COM.ELEMDESC();
```

Comment:
Can we use modern syntax:
```suggestion
                COM.ELEMDESC ElementDescription = new();
```